.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 // Place your settings in this file to overwrite default and user settings.
 {
   "[python]": {
-    "editor.defaultFormatter": "charliermarsh.ruff"
+    "editor.defaultFormatter": "ms-python.black-formatter"
```

Comment:
why? we use ruff in CI

Reply:
Not on purpose

Reply:
I don't have ruff so this setting completely destroys my local development

Reply:
Are you hoping to avoid installing it? I would argue for using the same thing we use in CI

Reply:
I have never even looked into what ruff or how to install it. I've been busy with other stuff :p

Reply:
It's great, imo! Give it a try. Much faster replacement for the `flake8` linter that now also does good formatting (replacing black)

Reply:
I'm running it since a few days back :)